chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/presenter/CertifiedProductCsvPresenter.java

```diff
@@ -147,18 +148,21 @@ public void setFf4j(FF4j ff4j) {
                 : listing.getDeveloper().getWebsite());
         result.add(formatSelfDeveloper(listing));
         result.addAll(getContactCells(listing));
-        result.add(listing.getProduct().getName());
-        result.add(listing.getProduct().getProductId().toString());
-        result.add(listing.getVersion().getVersion());
-        result.add(listing.getVersion().getVersionId().toString());
+        result.add(listing.getProduct() != null ? listing.getProduct().getName() : "?");
```

Comment:
What about using a constant for the `?` at the end of this ternary? That way, if we _do_ need to change it for some reason it'll be easy to do it in one place. Especially if this ends up becoming something that could be seen by the public